src/containers/views/Documents.js

```diff
@@ -98,12 +98,19 @@ export class Documents extends Component {
       return null;
     }
 
+    let newBtn = null;
+    if (collection_name == "posts") {
+      newBtn = "New Post";
+    } else {
+      newBtn = "New Document";
+    }
+
     return (
       <div>
         <div className="content-header">
           <h1>{capitalize(collection_name)}</h1>
           <div className="page-buttons">
-            <Link className="btn btn-active" to={`${ADMIN_PREFIX}/collections/${collection_name}/new`}>New post</Link>
+            <Link className="btn btn-active" to={`${ADMIN_PREFIX}/collections/${collection_name}/new`}>{newBtn}</Link>
```

Comment:
If you could add a test [here](https://github.com/jekyll/jekyll-admin/blob/master/src/containers/views/tests/documents.spec.js#L35) for this, it would be great. I'd be glad to help you.